conf/defaults.ini

```diff
@@ -26,6 +26,17 @@ plugins = data/plugins
 # folder that contains provisioning config files that grafana will apply on startup and while running.
 provisioning = conf/provisioning
 
+#################################### Provisioning ###############################
+[provisioning]
+# folder that contains datasource provisioning config files, default is paths.provisioning/datasources
```

Comment:
The style guide dictates it should be ["data source"](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#data-source):
```suggestion
# folder that contains data source provisioning config files, the default is paths.provisioning/datasources
```